rental.rb

```diff
@@ -1,15 +1,23 @@
-require './book'
-require './person'
-
 class Rental
-  attr_accessor :date, :book, :person
+  attr_accessor :date, :book, :person, :rentals
+  attr_reader :id
 
-  def initialize(date, book, person)
+  def initialize(person, book, date)
     @date = date
+
     @person = person
+    person&.rentals&.push(self)
```

Comment:
- [ ] @DJ-MrJay am getting the error below when I run the app, kindly check the functionality of this method since the error is indicating it as the source
![image](https://user-images.githubusercontent.com/61408860/213734423-2e883d03-af5f-4447-a937-86d4279af855.png)